samples/custom_dispatch/cpu/mlp_plugin/mlp_all_specs.pdll

```diff
@@ -0,0 +1,119 @@
+#include "mlir/Dialect/Linalg/IR/LinalgOps.td"
+#include "mlir/Dialect/Arith/IR/ArithOps.td"
+#include "mlir/Dialect/Func/IR/FuncOps.td"
+#include "mlir/Dialect/Tensor/IR/TensorOps.td"
+#include "mlir/Dialect/Tosa/IR/TosaOps.td"
+#include "torch-mlir/Dialect/Torch/IR/TorchOps.td"
+#include "torch-mlir/Dialect/TorchConversion/IR/TorchConversionOps.td"
```

Comment:
Can we avoid mixing TOSA and Torch in a single file?

Reply:
Done, includes have been removed and each rewrite is split into its individual file based on dialect